agent/src/endpoints.ts

```diff
@@ -116,6 +118,7 @@ const zCreatePresentationRequestBody = z.object({
 
 apiRouter.post('/requests/create', async (request: Request, response: Response) => {
   const verifier = await getVerifier()
+  // This will also be done for the C flow, but that does not matter right now
```

Comment:
would be very easy to just provide the flow in the request.body right? I mean it's ok ,but also not really a reason to not handle it?